src/library-authoring/common/specs/OrganizationDropdown.spec.jsx

```diff
@@ -0,0 +1,110 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import { act } from 'react-dom/test-utils';
+import OrganizationDropdown from '../OrganizationDropdown';
+
+const initFireEvent = () => {
+  const map = {};
+
+  document.addEventListener = jest.fn((event, cb) => {
+    map[event] = cb;
+  });
+
+  document.removeEventListener = jest.fn(event => {
+    delete map[event];
+  });
+
+  return map;
+};
+
+describe('common/OrganizationDropdown.jsx', () => {
+  let props;
+
+  beforeEach(() => {
+    props = {
```

Comment:
It is better to put `const props = ...` in the start of the file and remove `beforeEach` in this case